pkg/metadata/host/host_linux.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package host
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/network/config/sysctl"
+)
+
+// GetNetworkInfo returns host specific network configuration.
+// At this time, only information queried is the ephemeral port range
+func GetNetworkInfo() (*NetworkInfo, error) {
+	intpair := sysctl.NewIntPair("/proc", "net/ipv4/ip_local_port_range", 0)
```

Comment:
I think we have a configuration / env value for the proc root that we should use.